keyboards/enmeseoz/enmepad3x4/keymaps/default/keymap.c

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2025 ENMESEOZ
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    LAYOUT_3x4(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
```

Comment:
Only the 3rd and 4th layer were previously requested to be removed.
```suggestion
        KC_0,    KC_1,    KC_4,    KC_7, 
        KC_ENT,  KC_2,    KC_5,    KC_8, 
        MO(1),   KC_3,    KC_6,    KC_9
```